src/main/java/qna/domain/Answer.java

```diff
@@ -3,13 +3,21 @@
 import qna.NotFoundException;
 import qna.UnAuthorizedException;
 
+import javax.persistence.*;
 import java.util.Objects;
 
-public class Answer {
+@Entity
+public class Answer extends BaseEntity {
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
```suggestion
    @Id 
    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
위 방법과 한 라인에 모두 작성되는 방식 모두 괜찮다고 생각합니다! 
다만 어느 한 방법으로 통일 되는 것이 좋을 것 같아요! 😄

Reply:
그러네요. 인텔리제이에서 자동으로 먹히는데 이건 자동으로 안되었나 보네요ㅠ